docs/Extensibility_UG/frontend_extensions.md

```diff
@@ -78,26 +78,48 @@ In the resulting converted OpenVINO model, “MyRelu” operation will be replac
+As described above, ``OpExtension`` is useful when attributes can be mapped one by one or initialized by a constant.
+Attributes in OpenVINO operators are identified by their names, so for frameworks that also have named attributes (like Tensorflow, PaddlePaddle, ONNX), you can specify name to name mapping. For frameworks where OpenVINO operator's attributes can be mapped to one of the framework operator inputs (like Pytorch), there's a name to input index mapping.
+
+Named attributes mapping
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the set of attributes in framework representation and OpenVINO representation completely match by their names and types, no attribute mapping has to be specified in OpExtension constructor parameters. The attributes are discovered and mapped automatically based on ``visit_attributes`` method that should be defined for any OpenVINO operation.
 
 Imagine you have CustomOperation class implementation that has two attributes with names ``attr1`` and ``attr2``:
 
 .. doxygensnippet:: docs/snippets/ov_extensions.cpp
    :language: cpp
    :fragment: [frontend_extension_CustomOperation]
 
-And original model in framework representation also has operation with name “CustomOperatoin” with the same ``attr1`` and ``attr2`` attributes. Then with the following code:
+And original model in framework representation also has operation with name ``CustomOperation`` with the same ``attr1`` and ``attr2`` attributes. Then with the following code:
 
 .. doxygensnippet:: docs/snippets/ov_extensions.cpp
    :language: cpp
    :fragment: [frontend_extension_CustomOperation_as_is]
 
-both ``attr1`` and ``attr2`` are copied from framework representation to OpenVINO representation automatically. If for some reason names of attributes are different but values still can be copied “as-is” you can pass attribute names mapping in ``OpExtension`` constructor:
+both ``attr1`` and ``attr2`` are copied from framework representation to OpenVINO representation automatically.
+
+The code is slightly different, if you want to use ``OpExtension`` for PaddlePaddle model. PaddlePaddle operations have named inputs and outputs, so you need to pass their names in ``OpExtension`` constructor. For example, if ``CustomOperation`` has 3 inputs named "A", "B" and "C" and two outputs named "X" and "Y", then the code that registers the extension looks like following:
+
+.. doxygensnippet:: docs/snippets/ov_extensions.cpp
+   :language: cpp
+   :fragment: [frontend_extension_CustomOperation_as_is_paddle]
+
+
+If for some reason names of attributes are different but values still can be copied “as-is” you can pass attribute names mapping in ``OpExtension`` constructor:
 
 .. doxygensnippet:: docs/snippets/ov_extensions.cpp
    :language: cpp
    :fragment: [frontend_extension_CustomOperation_rename]
 
```

Comment:
Here we should also describe Named indexed outputs for TensorFlow extensions.
@mvafin, will you add it here please?

Reply:
Please create a ticket for this

Reply:
Created - 108007